### PR TITLE
Add function to find gcd

### PR DESCRIPTION
**Pull request by udbhav-chugh**

### CODE REVIEW

File: cpp/number_theory/gcd.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+int gcd(int a, int b) 
+{
```
Comment (sahilbansal17) [inline cpp/number_theory/gcd.cpp:2]: Please follow indentation style

File: cpp/number_theory/gcd.cpp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+int gcd(int a, int b) 
+{ 
+    if (b==0)
```
Comment (sahilbansal17) [inline cpp/number_theory/gcd.cpp:3]: Give proper spacing here in condition

File: cpp/number_theory/gcd.cpp [hunk 3]
```diff
@@ -0,0 +1,7 @@
+int gcd(int a, int b) 
+{ 
+    if (b==0) 
+        return a; 
+    return gcd(b, a%b);
```
Comment (sahilbansal17) [inline cpp/number_theory/gcd.cpp:5]: Similar `a % b`

**Comment by udbhav-chugh** [comment]
> Please add requested changes :)

I have created a new pull request as I had deleted the previous forked version of the repo. So I could not update this pull request. You can close it without merging. Thank you.

**Pull request closed without merging**